superset-frontend/src/SqlLab/components/AceEditorWrapper/index.tsx

```diff
@@ -35,11 +35,17 @@ type HotKey = {
   func: (aceEditor: IAceEditor) => void;
 };
 
+type CursorPosition = {
+  row: number;
+  column: number;
+};
+
```

Comment:
Let's reuse the CursorPosition type you already defined in SqlLab/types.

```suggestion
import type { CursorPosition } from 'src/SqlLab/types';
```